src/test/java/com/epam/tc/hw4/page/BasePage.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.tc.hw4.page;
+
+import com.epam.tc.hw3.util.PropertiesUtil;
+import com.epam.tc.hw4.util.AttachmentUtil;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import io.qameta.allure.Step;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.ITestContext;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public abstract class BasePage {
+    protected WebDriver driver;
+    public com.epam.tc.hw3.util.PropertiesUtil propUtil = new PropertiesUtil();
```

Comment:
you already have an import for Property class, so you can remove com.epam.tc.hw3.util, it would be more readable